xe-daemon/xe-daemon.go

```diff
@@ -125,6 +128,14 @@ func main() {
 					}
 				}
 			}
+			if count%DivisorMax == 0 {
```

Comment:
I think we will detect the change in each iteration and why we choose the max to the update?